nanoCAT/recipes/coordination_number.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
```

Comment:
As a general rule of thumb:
* Builtin modules should be imported first.
* Third-party imports come second.
* Imports from within this very package last.